### ISSUE TITLE
Literal strings should be assignable to `Sequence[Unknown]`

### ISSUE THREAD

**Issue by sharkdp** [astral-sh/ty#531]
Found this while analyzing common ecosystem false positives. I think this should work?
```py
from typing import Sequence, Any

seq: Sequence[Any] = "abc"  #  `Literal["abc"]` is not assignable to `Sequence[Any]`
```
https://play.ty.dev/1059069f-082c-4e2f-a643-92b3f0d8583c

Both of these cases work:
```py
seq: Sequence[Any] = some_str
seq: Sequence[str] = "abc"
```

**Comment by MatthewMckee4** [comment]
Should literal strings be subtypes of `Sequence[Unknown]`?

**Comment by AlexWaygood** [comment]
nothing should be a subtype of `Sequence[Unknown]` because it's not a fully static type; only fully static types participate in subtyping

**Comment by MatthewMckee4** [comment]
ah yeah sorry, thanks

**Comment by MatthewMckee4** [comment]
Okay, ill submit a PR for this
